src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -64,6 +64,8 @@ public class EditCommand extends Command {
     public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Person: %1$s";
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
     public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book.";
+    public static final String MESSAGE_HAS_CLASHES = "\n You have %d other students with clashing schedule:\n%s";
```

Comment:
Just a question because I am not very sure: Will this lead to there being an extra space being displayed in the UI before the "You"?

Reply:
Fixed. Removed the space